map/bookmark_manager.cpp

```diff
@@ -1841,15 +1841,15 @@ void BookmarkManager::LoadBookmarkRoutine(std::string const & filePath, bool isT
     auto collection = std::make_shared<KMLDataCollection>();
 
     // Convert KML/KMZ/KMB files to temp KML file and GPX to temp GPX file.
-    std::string fileSavePath = GetKMLorGPXPath(filePath);
-    if (!fileSavePath.empty())
+    std::vector<std::string> filesToLoad = GetKMLOrGPXFilesPathsToLoad(filePath);
+    for (auto const & fileToLoad : filesToLoad)
```

Comment:
A const is missing. But better do like this (it limits the scope of the var to a loop):
```suggestion
    for (auto const & fileToLoad : GetKMLOrGPXFilesPathsToLoad(filePath))
```

Reply:
Fixed